mlflow/genai/judges/instructions_judge/constants.py

```diff
@@ -0,0 +1,33 @@
+"""
+Constants for the InstructionsJudge module.
+
+This module contains constant values used by the InstructionsJudge class,
+including the augmented prompt template for trace-based evaluation.
+"""
+
+# Augmented prompt template for trace-based evaluation
+INSTRUCTIONS_JUDGE_TRACE_PROMPT_TEMPLATE = """
```

Comment:
@BenWilson2 Can we remove this constant & remove this file until we add support for `make_judge().__call__()` on traces?